signac/__main__.py

```diff
@@ -133,15 +133,15 @@ def _open_job_by_id(project, job_id):
         )
 
 
-def find_with_filter_or_none(args):
+def _find_with_filter_or_none(args):
```

Comment:
I'm fine with making all of these very explicitly internal, but I also don't think it's critical in a module named `__main__.py` that is clearly not part of the public API.

Reply:
I agree — the other helpers in the main module already began with an underscore so I figured it was a helpful change for consistency. There was no difference in intended visibility so we should just make all of them private.